ui/src/app/views/account-creation-success/component.ts

```diff
@@ -34,4 +36,8 @@ export class AccountCreationSuccessComponent {
   signIn(): void {
     this.signInService.signIn();
   }
+
+  routeToTFA(): void {
+    window.location.href = 'https://myaccount.google.com/security';
```

Comment:
Any reason not to use a normal link instead of a click callback? (`<a href=...`). In general I prefer links as they tend to integrate better with browsers, e.g. control+click, or hovering showing the target URL. You can also do a link+button; not sure if you have examples of doing this in the codebase already.

Reply:
Using href is probably better. There's a lot of changes actually needing to happen to this PR (given some feedback/discussion) so I'm actually going to close it out for now, and reopen when that's been done.